requirements.txt

```diff
@@ -0,0 +1,20 @@
+astroid==2.4.2
```

Comment:
We are using poetry not pip so a requirements file is not required.

Add dependencies to your `pyproject.toml` using commands like `poetry add requests`.

Reply:
Hi, I'm not sure when that requirements file came into being? Maybe when I installed requests?

Reply:
Removed it and it done the poetry thing.